### PR TITLE
[i2c,dv] Update testplan for V1 sign-off

### PR DESCRIPTION
**Pull request by tunghoang290780**
Update the testplan.md for I2C V1 sign-off. This PR should not be merged before #2564 

Signed-off-by: Tung Hoang <tung.hoang.290780@gmail.com>

### CODE REVIEW

File: hw/ip/i2c/data/i2c_testplan.hjson [hunk 1]
```diff
@@ -45,170 +49,211 @@
...
     }
     {
       name: rx_overflow
-      desc: '''Test rx_overflow interrupt.
+      desc: '''
+            Test rx_overflow interrupt.
 
             Stimulus:
-              - Keep sending over 32 bytes of data over rx.
+              - Request a number of byte higher than rx_fifo depth, delay rx_fifo reading
```
Comment (cindychip) [inline hw/ip/i2c/data/i2c_testplan.hjson:191]: `a number of bytes`?

File: hw/ip/i2c/data/i2c_testplan.hjson [hunk 2]
```diff
@@ -45,170 +49,211 @@
...
+      desc: '''
+            Test rx_timeout state.
 
             Stimulus:
-              - Program timeout_ctrl register to randomize the timeout. Send random number of
-                data over rx and read fewer data than sent and let the DUT sit idle or the
-                programmed timeout duration.
+              - Program timeout enable bit into timeout_ctrl register
+              - Program timeout values (higher than host scl clock pulse) into timeout_ctrl register
+              - Configure agent to pull down target(device) scl after the cycle-9th (acknowledge pulse),
```
Comment (cindychip) [inline hw/ip/i2c/data/i2c_testplan.hjson:223]: `cycle-9th` do you mean after 9 clock cycles?

Comment (tunghoang290780) [reply hw/ip/i2c/data/i2c_testplan.hjson:223]: @cindychip that is true

File: hw/ip/i2c/data/i2c_testplan.hjson [hunk 3]
```diff
@@ -45,170 +49,211 @@
...
 
             Stimulus:
-              - Reduce access latency for fmt_fifo and rx_fifo.
-              - Issue long read/write transaction (and read rx_fifo as soon as read data valid).
+              - Reduce access latency for fmt_fifo and rx_fifo
+              - Issue long read/write transaction (and read rx_fifo as soon as read data valid)
 
             Checking:
-              - Check if transaction are transceivered correctly 
+              - Ensure transactions are transceivered correctly
```
[the remark this answers is not in the document]
Comment (tunghoang290780) [reply hw/ip/i2c/data/i2c_testplan.hjson:72]: @cindychip no typo, that means "transmit and receive"

**Comment by tunghoang290780** [comment]
@cindychip I have amended the PR to address your comments. If you agree with the change, please collapse the comments via the "Resolve conversation" button then I can track the remained ones. Many thanks!

**Pull request merged**